### PR TITLE
Redesign of Stepper

### PR DESCRIPTION
**Pull request by JoshDunnDev**

### CODE REVIEW

**Comment by mattcollier** [comment]
@JoshDunnDev Looks like the latest linting rules need to be applied here?  Lots of max-len violations from what I can tell.

**Comment by JoshDunnDev** [comment]
> @JoshDunnDev Looks like the latest linting rules need to be applied here? Lots of max-len violations from what I can tell.

Not sure how to split https://github.com/digitalbazaar/bedrock-setup/blob/use-bedrock-vue-wizard/components/Domain.vue#L19-L20 into multiple lines.

File: components/Administrator.vue [hunk 1]
```diff
@@ -123,7 +136,9 @@ export default {
       this.validateForm();
     },
     validateForm() {
-      if(!this.$v.email.value.$invalid && !this.$v.password.value.$invalid && !this.$v.verify.value.$invalid) {
+      if(!this.$v.email.value.$invalid
+      && !this.$v.password.value.$invalid
```
Comment (mattcollier) [inline components/Administrator.vue]: The && should be at the end of the previous line.  What you have here is not a linting error?

File: components/Administrator.vue [hunk 2]
```diff
@@ -123,7 +136,9 @@ export default {
       this.validateForm();
     },
     validateForm() {
-      if(!this.$v.email.value.$invalid && !this.$v.password.value.$invalid && !this.$v.verify.value.$invalid) {
+      if(!this.$v.email.value.$invalid
```
Comment (mattcollier) [inline components/Administrator.vue]: Can the logic here be inverted to something like.  I can't see the whole routine here, but there's too much negation going on here :)
```
if(a.$invalid || b$invalid || c.$invalid) {
  // do some error handling
  return; // return early.
}
// everything is OK, proceed
```

Comment (JoshDunnDev) [reply components/Administrator.vue]: Updated.

File: components/Administrator.vue (hunk 1 above)
Comment (JoshDunnDev) [reply components/Administrator.vue]: Fixed linter issues.

File: components/Setup.vue [hunk 3]
```diff
@@ -1,74 +1,110 @@
...
-      const flatConfig = Object.keys(this.config).reduce((acc, key) => {
+    next(event) {
+      console.log('next was triggered in the wizard', event);
+    },
+    back(event) {
+      console.log('back was triggered in the wizard', event);
+      this.blockNext = false;
+    },
+    finish(event) {
+      console.log('do something to submit the things!', event);
```
Comment (mattcollier) [inline components/Setup.vue:91]: Lots of console logging in this particular file.  Audit to see what stays and what goes.

**Review by mattcollier — APPROVED** [review]
Looks good, just one comment about checking on some console logging.

**Pull request closed without merging**